pages/dashboard/events.tsx

```diff
@@ -41,87 +37,106 @@ function Events({ designation }: any) {
-                  eventResource[eventResource.length - 1].length === 0
-                    ? `/assets/logos/${eventResource[0]}.png`
-                    : eventResource.slice(-1)[0]
-                }
-                startDate={activeEvent?.start ? activeEvent.start : new Date()}
-                endDate={activeEvent?.end ? activeEvent.end : new Date()}
-              />
-            ) : (
-              <></>
-            )}
-
-            <Calendar
-              localizer={localizer}
-              events={listedEvents}
-              className={styles.calendar}
-              popup={true}
-              onSelectEvent={handleSelectEvent}
-              startAccessor={(event) => new Date(event.start ?? Date.now())}
-              endAccessor={(event) => new Date(event.end ?? Date.now())}
-              eventPropGetter={getEventClassByEvent}
-              views={{
-                month: true,
-                week: false,
-                day: false,
-                agenda: false,
-              }}
-              dayLayoutAlgorithm={"no-overlap"}
-            />
-          </div>
-        </div>
-      </DashboardLayout>
+        <DashboardLayout title="Events | ACM at PEC" heading="Events">
+            <div className={styles.events}>
+                <p>Have a look at our calendar</p>
+
+                <div className={styles.calendarWrapper}>
+                    {designation === "Admin" ||
+                    designation === "Core" ||
+                    designation === "ExecutiveBody" ? (
+                        <div className={styles.eventButton}>
+                            <button type="submit" onClick={handleCreateEvent}>
+                                Create Event
+                            </button>
+                        </div>
+                    ) : (
+                        <></>
+                    )}
+
+                    {createModal ? (
+                        <CreatePopup
+                            open={createModal}
+                            setOpen={setCreateModal}
+                            appendEvent={appendEvent}
+                        />
+                    ) : (
+                        <></>
+                    )}
+
+                    {showModal ? (
+                        <EventPopup
+                            handleClose={() => {
+                                setShowModal(false);
+                            }}
+                            title={activeEvent?.title as string}
+                            subTitle={eventResource[1]}
+                            description={eventResource.slice(
+                                2,
+                                eventResource.length - 1
+                            )}
+                            imageUrl={
+                                eventResource[eventResource.length - 1]
+                                    .length === 0
+                                    ? `/assets/logos/${eventResource[0]}.png`
```

Comment:
`eventResource[0]` is branch name, originally they matched file name `dev.png`, `ai.png` etc but now  `dev` has been renamed to `DEVELOPMENT`. 
> To be tested in future >__<

Reply:
yeah, this is a bug, fixed now 👍